### DISCUSSION TITLE
feature: add last boolean query param to GET workflow /runs

### DISCUSSION THREAD

**Discussion by mxfactorial** [community/community#119878] [Actions]
### Select Topic Area

General

### Body

the async https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#create-a-workflow-dispatch-event doesnt return a run id

please add a `last` query param to https://docs.github.com/en/rest/actions/workflow-runs?apiVersion=2022-11-28#list-workflow-runs-for-a-workflow to avoid [querying by date](https://github.com/systemaccounting/mxfactorial/blob/bc252324052655e4642526f1f892375d87536b8f/scripts/build-all-images.sh#L51-L55)

**Comment by grandmas-favorite** [comment]
Thanks for sharing @mxfactorial! You can check our [public roadmap](https://github.com/github/roadmap/projects/1) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.

While it has been a while since we have updated on this thread, the absence of an update is simply that - there has been no update because there is so far nothing to report. That doesn't mean it isn't being worked on - it might simply be that we're unable to share details about work that isn't ready for release.

As a GitHub user, the most you can do is to share your feedback on this Discussion space with the correct tags (include examples), and trust that we are always working to make GitHub as awesome as possible. We can't guarantee that every bit of feedback sees action, but we are paying attention.
